CHANGELOG.md

```diff
@@ -48,6 +48,12 @@ _None._
 
 _None._
 
+## [5.2.0](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/releases/tag/5.0.0)
```

Comment:
```suggestion
## [5.2.0](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/releases/tag/5.2.0)
```

Reply:
Ops, copy-and-paste error, thanks for fixing it 😄